dockers/docker-orchagent/orchagent.sh

```diff
@@ -30,6 +30,8 @@ elif [ "$HWSKU" == "INGRASYS-S8900-54XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 elif [ "$HWSKU" == "INGRASYS-S8900-64XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
+elif [ "$HWSKU" == "Accton-AS7712-32X" ]; then
```

Comment:
after we revise the value of $HWSKU, according to your porting guide, should we revise `<HARDWARE_SKU>` accordingly?

device/
|-- <VENDOR_NAME>/
|   |-- <ONIE_PLATFORM_STRING>/
|   |   |-- <HARDWARE_SKU>/
|   |   |   |-- port_config.ini
|   |   |   |-- sai.profile
|   |   |-- plugins/
|   |   |   |-- eeprom.py
|   |   |   |-- led_control.py
|   |   |   |-- sfputil.py
|   |   |-- fancontrol
|   |   |-- installer.conf